test/dotnet-list-proj.Tests/GivenDotnetListProj.cs

```diff
@@ -0,0 +1,141 @@
+// Copyright (c) .NET Foundation and contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using FluentAssertions;
+using Microsoft.DotNet.Cli.Sln.Internal;
+using Microsoft.DotNet.Tools.Test.Utilities;
+using System;
+using System.IO;
+using System.Linq;
+using Xunit;
+
+namespace Microsoft.DotNet.Cli.List.Proj.Tests
+{
+    public class GivenDotnetListProj : TestBase
+    {
+        [Theory]
+        [InlineData("--help")]
+        [InlineData("-h")]
+        public void WhenHelpOptionIsPassedItPrintsUsage(string helpArg)
+        {
+            var cmd = new DotnetCommand()
+                .ExecuteWithCapturedOutput($"list projects {helpArg}");
+            cmd.Should().Pass();
+            cmd.StdOut.Should().Contain("Usage");
+        }
+
+        [Fact]
+        public void WhenTooManyArgumentsArePassedItPrintsError()
+        {
+            var cmd = new DotnetCommand()
+                .ExecuteWithCapturedOutput("list one.sln two.sln three.sln projects");
+            cmd.Should().Fail();
+            cmd.StdErr.Should().Contain("Unrecognized command or argument");
+        }
+
+        [Theory]
+        [InlineData("idontexist.sln")]
+        [InlineData("ihave?invalidcharacters.sln")]
+        [InlineData("ihaveinv@lidcharacters.sln")]
+        [InlineData("ihaveinvalid/characters")]
+        [InlineData("ihaveinvalidchar\\acters")]
+        public void WhenNonExistingSolutionIsPassedItPrintsErrorAndUsage(string solutionName)
+        {
+            var cmd = new DotnetCommand()
+                .ExecuteWithCapturedOutput($"list {solutionName} projects");
+            cmd.Should().Fail();
+            cmd.StdErr.Should().Contain("Could not find");
```

Comment:
Again, feels like not enough. Why aren't we doing something like Could not find solutionName?